### PR TITLE
switch rsa provider for full framework, to avoid not supported exception

### PR DESCRIPTION
**Pull request by r22016**
RSA.Create returning a implementation not supported on windows, switched to using RSACryptoProvider for full framework.

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@r22016__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (rameshr). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by markcowl** [comment]
@r22016 This doesn't seem to impact the test coverage.  Is there a test for this scenario, if not, please add one, or explain why that is not possible.

**Review by markcowl — CHANGES_REQUESTED** [review]
Please add test coverage for the change

**Comment by markcowl** [comment]
@vkc12uec Can you verify this PR and respond about testing?

**Comment by shahabhijeet** [comment]
@azuresdkci test this please

File: src/ResourceManagement/ServerManagement/Microsoft.Azure.Management.ServerManagement/Utility.cs [hunk 1]
```diff
@@ -29,8 +29,8 @@ public static string EncryptUsingGatwewaySettings(GatewayStatus status, string s
             var result = string.Empty;
             var jwk = status.EncryptionJwk;
 
-#if PORTABLE
-            using (var rsa = RSA.Create())
+#if NET45
```
Comment (shahabhijeet) [inline src/ResourceManagement/ServerManagement/Microsoft.Azure.Management.ServerManagement/Utility.cs]: Can we make something like below, this way we do not lose the existing Else part code.
#if NET45
#elseIf Portable
#else

**Review by shahabhijeet — COMMENTED** [review]
@vkc12uec can you please comment if losing the #else part of the code (proposed change) is fine with you.

File: src/ResourceManagement/ServerManagement/Microsoft.Azure.Management.ServerManagement/Utility.cs (hunk 1 above)
Comment (r22016) [inline src/ResourceManagement/ServerManagement/Microsoft.Azure.Management.ServerManagement/Utility.cs]: switched to using above pattern for preprocessor directive
updated to latest swagger spec to include support for new installerDownload property and related test

**Review by markcowl — APPROVED** [review]

**Comment by markcowl** [comment]
@rss016 please update the library version in project.json and in AssemblyInfo.cs

File: src/ResourceManagement/ServerManagement/Microsoft.Azure.Management.ServerManagement/Generated/Models/GatewayExpandOption.cs [hunk 2]
```diff
@@ -16,6 +16,8 @@ namespace Microsoft.Azure.Management.ServerManagement.Models
     public enum GatewayExpandOption
     {
         [System.Runtime.Serialization.EnumMember(Value = "status")]
-        Status
```
Comment (markcowl) [inline src/ResourceManagement/ServerManagement/Microsoft.Azure.Management.ServerManagement/Generated/Models/GatewayExpandOption.cs]: You will need to update the version in project.json and assemblyinfo.cs to release this change.

**Review by markcowl — COMMENTED** [review]
Need to update versions, otherwise LGTM

File: src/ResourceManagement/ServerManagement/Microsoft.Azure.Management.ServerManagement/Generated/Models/GatewayExpandOption.cs (hunk 2 above)
Comment (r22016) [inline src/ResourceManagement/ServerManagement/Microsoft.Azure.Management.ServerManagement/Generated/Models/GatewayExpandOption.cs]: updated version info

**Pull request merged**